content/en/docs/ambient/getting-started/_index.md

```diff
@@ -57,12 +57,9 @@ You can verify the installed components using the command `istioctl verify-insta
 
 ## Install the Kubernetes Gateway API CRDs
 
-You need to install the Kubernetes Gateway API CRDs, which don’t come installed by default on most Kubernetes clusters:
+You need to install the [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) CRDs, which don’t come installed by default on most Kubernetes clusters:
```

Comment:
the partial already includes

> Note that the Kubernetes Gateway API CRDs do not come installed by default on most Kubernetes clusters, so make sure they are installed before using the Gateway API:

so (a) we don't need both (b) I think we should change the text in the partial. A tip callout?  Just "The Kubernetes Gateway API is packaged as CRDs, which are not installed by default in most Kubernetes clusters. Install or upgrade them before using:"  ?

Reply:
No we can use the partial, I removed most of the dupe headers but I missed this one.